diff --git a/src/libxls/xls.h b/src/libxls/xls.h
index 89b8b92..fb5eab21 100644
--- a/src/libxls/xls.h
+++ b/src/libxls/xls.h
@@ -33,7 +33,15 @@
 
 #ifndef XLS_INCLUDE
 #define XLS_INCLUDE
- 
+
+/* readxl modification: process these headers in the global namespace, not inside
+ * namespace xls. Otherwise, on libcs (e.g. musl/Alpine) where these headers have
+ * not been transitively pre-loaded by other system or vendored headers, their
+ * typedefs (int64_t, gid_t, uid_t etc.) end up trapped in xls:: when libxls is
+ * consumed from C++ code. See readxl issues #775 and #790. */
+#include <stdint.h>
+#include <sys/types.h>
+
 #ifdef __cplusplus
 namespace xls {
 extern "C" {
