diff --git a/src/libxls/endian.c b/src/libxls/endian.c
index f126936..3c41050 100644
--- a/src/libxls/endian.c
+++ b/src/libxls/endian.c
@@ -35,7 +35,7 @@
 #include "libxls/endian.h"
 #include "libxls/ole.h"
 
-int xls_is_bigendian()
+int xls_is_bigendian(void)
 {
 #if defined (__BIG_ENDIAN__)
     return 1;
diff --git a/src/libxls/locale.c b/src/libxls/locale.c
index 8f8eb35..8dcdcce 100644
--- a/src/libxls/locale.c
+++ b/src/libxls/locale.c
@@ -47,7 +47,7 @@
     static char* old_locale;
 #endif
 
-xls_locale_t xls_createlocale() {
+xls_locale_t xls_createlocale(void) {
 #if defined(WINDOWS_BEFORE_RTOOLS_42)
     old_locale = setlocale(LC_CTYPE, ".65001");
     return NULL;
